install/docker.sh

```diff
@@ -1,15 +1,24 @@
-sudo apt install -y docker.io docker-buildx
+# Remove conflicting Docker packages
+sudo apt purge --auto-remove docker.io docker-buildx docker-doc docker-compose docker-compose-v2 podman-docker containerd runc
```

Comment:
None of these packages should be installed on a default 24.04, though, right? We haven't hardened anything else against stuff like that. Think we should just base this off the fresh setup, yeah?

Reply:
Yes, I don't think any of these would be installed by default. I know we list these in our docs to make users aware that if they have them installed, they may conflict.

But if we assume "fresh install" it's probably better to remove this part. If the packages have the right "conflicts" they should still make the installation fail (which is time for the user to read the error and fix the problem 🥹😅)

Reply:
Cool to target fresh setups only. Should uninstall scripts target current Omakub only, then, or previous versions as well?

Reply:
Would just target current version. Uninstall is just a nice-to-have anyway. You can always remove excess stuff by hand.